ch(authentication): Authenticate POST /loans

ibkadeeko opened the pull request:
#### What does this PR do?
Authenticate POST /loans route

#### Description of Task to be completed
- Created Verify Token Middleware
- Created Admin Seed table method
- Authenticated POST /loans route

#### How should this be manually tested?
1. Clone this branch.
2. Run `npm install` to install the dependencies.
3. Run `npm test` to run the tests.

#### What are the relevant pivotal tracker stories?
[#166104628](https://pivotaltracker.com/story/show/166104628)

#### Screenshots (if appropriate)
N/A

ibkadeeko committed:
ch(authentication): Authenticate POST /loans

Created Verify Token Middleware
Created Admin Seed table method
Authenticated POST /loans route

[finishes #166104628]

luqmanoop commented on server/middleware/verifyToken.js:
```diff
@@ -0,0 +1,57 @@
+import dotenv from 'dotenv';
+import { errorRes } from '../utils/responseHandler';
+
+dotenv.config();
```
this line should only be called in your app entry file

ibkadeeko replied:
I've removed it

luqmanoop commented on server/middleware/verifyToken.js:
```diff
@@ -0,0 +1,57 @@
+      res.locals.email = email;
+      next();
+    } catch (error) {
+      return errorRes(next, 500, error.message);
```
Consider returning a custom response rather than sending the actual message back to the client.

ibkadeeko replied:
I've changed it

luqmanoop commented on server/middleware/verifyToken.js:
```diff
@@ -0,0 +1,57 @@
+      res.locals.email = email;
+      next();
+    } catch (error) {
+      return errorRes(next, 500, error.message);
```
Same thing here.

ibkadeeko replied:
Done.

luqmanoop reviewed (changes requested):
Please attend to the feedback I left you.

ibkadeeko committed:
bg(feedback): Implement LFA feedback

Return custom error messages to user
Changed status codes
refactored tests
Require dotenv only at app.js

luqmanoop commented:
Go ahead and merge
